tests/jerry/array-prototype-splice.js

```diff
@@ -175,3 +175,174 @@ try {
   assert (e.message === "foo");
   assert (e instanceof ReferenceError);
 }
+
+try {
+  Array.prototype.splice.call (undefined);
+  assert (false);
+} catch (e) {
+  assert (e instanceof TypeError);
+}
+
+try {
+  var o = {};
+  Object.defineProperty(o, 'toString', { 'get' : function () { throw new ReferenceError ("foo"); } });
+  var a = { length : o }
+  Array.prototype.splice.call(a, function () { })
+  assert (false);
+} catch (e) {
+  assert (e instanceof ReferenceError);
+}
+
+try {
+  var o = {};
+  Object.defineProperty(o, 'toString', { 'get' : function () { throw new ReferenceError ("foo"); } });
+  [1,2].splice(o)
+  assert (false);
+} catch (e) {
+  assert (e instanceof ReferenceError);
+}
+
+try {
+  var o = {};
+  Object.defineProperty(o, 'toString', { 'get' : function () { throw new ReferenceError ("foo"); } });
+  [1,2].splice(1, o)
+  assert (false);
+} catch (e) {
+  assert (e instanceof ReferenceError);
+}
+
+try{
```

Comment:
Missing space before `'{'`
There are many minor style issues like this in the added test file. Please always put a space before `'{'`, put semicolon after statements, do not write space before `'('` in function calls (e.g.: `assert(false);`, etc.)

Check your code and fix these issues.